src/KygekDev/AntiBedrock/AntiBedrock.php

```diff
@@ -14,14 +14,14 @@
 
 namespace KygekDev\AntiBedrock;
 
-use pocketmine\block\Block;
+use pocketmine\block\VanillaBlocks;
 use pocketmine\event\block\BlockBreakEvent;
 use pocketmine\event\Listener;
 use pocketmine\plugin\PluginBase;
 
 class AntiBedrock extends PluginBase implements Listener {
 
-    public function onEnable() {
+    public function onEnable() : void {
```

Comment:
Make it similar with `PluginBase`:

```suggestion
    protected function onEnable() : void {
```